### DISCUSSION TITLE
How to handle the case where i need data transformer after the DTO class is instanced?

### DISCUSSION THREAD

**Discussion by dpetrovaliev** [spatie/laravel-data#206] [Q&A] [answered]
I have a multi-select field on my website and use a DTO to organize the data transferring.
In the DB my field is a string but the front end gives me an array of strings.
I need to transform the data into a string while editing the field.
Data transformers and casts aren't fit for me because the field is being changed after the DTO is instanciated.

**Comment by Nir-An** [comment, marked as answer]
hi @dpetrovaliev 
if you are sure that [transformer](https://spatie.be/docs/laravel-data/v2/advanced-usage/creating-a-transformer ) isn't fit your case
 
You are always can do it manually : 
https://spatie.be/docs/laravel-data/v2/as-a-data-transfer-object/creating-a-data-object#content-magical-creation
